packages/indexer-cli/src/commands/indexer/actions/get.ts

```diff
@@ -107,6 +142,13 @@ module.exports = {
         throw Error(`Invalid value for '--first' option, must have a numeric value.`)
       }
 
+      if (!['undefined', 'string'].includes(typeof fields)) {
+        throw Error(
+          `Invalid value for '--fields' option, must be a comma-separated list of field names.`,
```

Comment:
For formatting consistency let's remove the `.` at the end here.  
(I just found a few other instances of periods used and removed them to maintain this consistent formatting across indexer-cli error messages).

Reply:
Fixed in bc4e412e3c0eaac221389aec50b4bdee7c42f376.